regex-assembly/exclude/unix-shell-fps-pl2.ra

```diff
@@ -17,12 +17,22 @@ aptitude~
 dnf
 dnf@
 dnf~
+more
+more@
+more{{no-space}}
```

Comment:
Remove the "definition" versions (`{{...}}`). They no longer exist.

Reply:
Thank you very much!